scipy/stats/stats.py

```diff
@@ -3692,6 +3692,103 @@ def f_oneway(*args, axis=0):
     return F_onewayResult(f, prob)
 
 
+AlexanderGovernResult = namedtuple("AlexanderGovernResult", ("statistic",
+                                                             "pvalue"))
+
+
+def AlexanderGovern(*args):
+    """
+    Performs the Alexander Govern test.
+
+    The Alexander-Govern approximation tests the equality of k independent
+    means in the face of heterogeneity of variance. The test is applied to
+    samples from two or more groups, possibly with differing sizes.
+
+    Parameters
+    ----------
+    sample1, sample2, ... : array_like
+        The sample measurements for each group.  There must be at least
+        two arguments.
+
```

Comment:
Would it be easy enough to support multidimensional arrays and have an axis parameter like [f_oneway](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.f_oneway.html#scipy.stats.f_oneway)

Reply:
I think it would - we can encorporate that in with the next revision.

Reply:
Let's add it in a separate PR after this is merged.